Lesson_12/Tusk_1.py

```diff
@@ -0,0 +1,87 @@
+import json
+
+
+print('Phone list program is started.')
+
+commands = ('You can use next commands:'
+            '\n  - stats: If you want to see the quantity of records'
+            '\n  - add: if you want to add a new record'
+            '\n  - delete <name>: if you want to delete a record with name in <>'
+            '\n  - list: to see the list of all records'
+            '\n  - show <name>: to see the information about the name in <>'
+            '\n  - commands: if you want to see tha command list again'
+            '\n  - exit: if you want to exit the program')
+print(commands)
+file_address = 'phone_book.txt'
+
+while True:
+    with open(file_address, 'r') as file:
```

Comment:
Move it outside the `while` loop. You need to load the data from a file only once when your program is started. Also, you can have an error here if a file does not exist, add some handling of such a case here.